app/src/main/AndroidManifest.xml

```diff
@@ -331,11 +331,6 @@
                     android:scheme="package"/>
             </intent-filter>
         </receiver>
-        <receiver android:name="de.danoeh.antennapod.core.service.playback.MediaButtonIntentReceiver">
-            <intent-filter>
-                <action android:name="android.intent.action.MEDIA_BUTTON" />
-            </intent-filter>
-        </receiver>
```

Comment:
`MediaButtonReceiver` is registered on the core module manifest.
I tested it on mine (nexus 5) with a watch and it works just like it used to.